src/anonfile/anonfile.py

```diff
@@ -46,15 +46,15 @@
 from tqdm import tqdm
 from urllib3 import Retry
 
-__version__ = "0.2.7"
+__version__ = "1.0.0"
```

Comment:
Oof, I had already made these changes, however I will try and see if I can merge our work conflict free. Thanks for being proactive though.